main/src/ca/uwaterloo/flix/language/ast/Ast.scala

```diff
@@ -378,9 +378,9 @@ object Ast {
   object Fixity {
 
     /**
-      * The atom is loose (it does not have to be fully materialized before it can be used).
+      * The atom is not fixed (it does not have to be fully materialized before it can be used).
       */
-    case object Loose extends Fixity
+    case object NotFixed extends Fixity
```

Comment:
Please undo this one :) I am still thinking of a good name. I am not satisfied with "NotFixed".

Reply:
Is unfixed better? hmm maybe that implies that is was previously fixed.. ill revert it